README.md

```diff
@@ -19,6 +19,13 @@
   <a href="https://join.slack.com/t/brasilapi/shared_invite/zt-l12s2b8k-r0SHGZV4YZSMfrzhydn8WA"><img src="https://files.readme.io/e23f0e0-Slack_RGB.png" width="160px"></a>
 </div>
 
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=BrasilAPI_BrasilAPI&metric=alert_status)](https://sonarcloud.io/dashboard?id=BrasilAPI_BrasilAPI)
+[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=BrasilAPI_BrasilAPI&metric=code_smells)](https://sonarcloud.io/dashboard?id=BrasilAPI_BrasilAPI)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=BrasilAPI_BrasilAPI&metric=coverage)](https://sonarcloud.io/dashboard?id=BrasilAPI_BrasilAPI)
```

Comment:
Essa badge do coverage tá com o texto "Measure has not been found"

Reply:
é que a primeira análise que ele rodou na master foi automática aí ele não pegou o coverage, [aqui](https://sonarcloud.io/dashboard?id=BrasilAPI_BrasilAPI&pullRequest=187) na análise da branch, ele conseguiu e quando fizer o merge vai ficar certo. Como todos os testes são e2e, o coverage tá 0.

![Captura de tela de 2021-01-18 10-37-14](https://user-images.githubusercontent.com/3614317/104922210-2913a400-5979-11eb-96f9-1c27fada91c9.png)